cinch/roles/jenkins_master/tasks/plugins.yml

```diff
@@ -56,7 +56,7 @@
 
 - name: perform plugin install
   jenkins_plugin:
-    url: "{{ _jenkins_url }}"
+    url: "http://localhost:8080"
```

Comment:
Cool, this option is not listed in the official docs, so I assumed it wouldn't work. But it does! Reverted.